src/components/pages/Step5.vue

```diff
@@ -13,6 +13,8 @@
                 <v-btn v-show="show_controls" :disabled="busy || invalid" @click="changeState">
                     {{ buttonCaption }}
                 </v-btn>
+                <p v-if="state === 'recording'" style="color: orange;">Videos are recording, do not refresh</p>
```

Comment:
@AlbertoCasasOrtiz, what is expected with the colors here?

Reply:
Initially, I was thinking of displaying errors and/or warnings during testing, but I decided against it since it would be redundant. We already show errors using toasts, which is consistent throughout the application, so it wouldn't make sense to display them there.

Basically, I forgot to remove the colors, I am going to remove it now.

Reply:
I just removed it.